src/main/java/dog/pawbook/model/managedentity/owner/Owner.java

```diff
@@ -19,7 +19,11 @@
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
 public class Owner extends Entity {
+    public static final String ADDRESS_PREFIX = "Address: ";
```

Comment:
I feel like the naming is can be quite confusing, I might have done something similar but on hindsight it is easily confused with the PREFIX_ADDRESS etc, perhaps make them private if not necessary to be exposed?